### DISCUSSION TITLE
Let code-server start at system startup (OS:Ubuntu)

### DISCUSSION THREAD

**Discussion by rumunius** [coder/code-server#3761] [Q&A]
My OS is Ubuntu 20.04.2 LTS.
My user name is `ubuntu`.
The code-server executable file is in the `/home/ubuntu/code-server-3.10.2-linux-amd64/bin/`.
How can I let code-server start at system startup?
I've tried many ways, but they don't work.

**Comment by nooobcoder** [comment]
@blackpancake try running
`sudo systemctl enable --now code-server@\$USER`

**Reply by rumunius** [reply]
After running that:
`Failed to enable unit: Unit file code-server@\x24USER.service does not exist.`
What should i do next?
